src/doc/gui_manual/MakingItPretty/Annotations.rst

```diff
@@ -816,3 +811,83 @@ an image transparent before VisIt_ displays it as an image annotation
 object, click on the **Transparent color** check box and then select a
 new color by clicking on the **Transparent color** button and picking a
 new color from the **Popup color menu**.
+
+Legend annotation objects
+"""""""""""""""""""""""""
+
+Legends are automatically added when plots are created, and have names that include the plot type.
+:numref:`Figure %s <legned_multiple_image>` shows multiple legends listed in the **Annotation Objects** tab.
+The Legend annotation object interface can be used to customize the legends position, size, tick labels and aspects of its appearance.
+
+.. _legend_multiple_image:
+
+.. figure:: images/legend_multiple_objects.png
+
+    Multiple legends in the **Annotation Objects** tab. 
+
+The **Position** tab, shown in :numref:`Figure %s <legend_position_image>` has controls for position, size and orientation.
+VisIt generally controls the positions of legends, but if you want a specific legend to be placed elsewhere in the visualization window, uncheck the **Let VisIt manage legend position** checkbox and modify the **Legend position**.
+The 2D coordinate used to position the legend matches the legend's lower left corner. (??? CHECK THIS!!!)
```

Comment:
Did you check this and if so, you should remove the "(??? CHECK THIS!!!)" comment.

Reply:
Another good catch, Thanks!  Verified and removed.